packages/create-llama/templates/index.ts

```diff
@@ -31,12 +31,18 @@ const createEnvLocalFile = async (
 
   switch (vectorDb) {
     case "mongo": {
+      content += `# For generating a connection URI, see https://www.mongodb.com/docs/guides/atlas/connection-string\n`;
       content += `MONGODB_URI=\n`;
       content += `MONGODB_DATABASE=\n`;
       content += `MONGODB_VECTORS=\n`;
       content += `MONGODB_VECTOR_INDEX=\n`;
       break;
     }
+    case "pg": {
+      content += `# For generating a connection URI, see https://www.postgresql.org/docs/16/libpq-connect.html#LIBPQ-CONNSTRING-URIS\n`;
+      content += `PG_CONNECTION_STRING=\n`;
+      break;
```

Comment:
similar to mongodb, we should be able to parameterize schema and table used to store the embeddings.
Looking at `PGVectorStore`, we need to update also that class.